[2024-03-16T09:53:28.647+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-03-16T08:00:00+00:00 [queued]>
[2024-03-16T09:53:28.652+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-03-16T08:00:00+00:00 [queued]>
[2024-03-16T09:53:28.652+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-16T09:53:28.673+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): checking_model_prediction_ratio> on 2024-03-16 08:00:00+00:00
[2024-03-16T09:53:28.678+0000] {standard_task_runner.py:60} INFO - Started process 603 to run task
[2024-03-16T09:53:28.681+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_scoring_inference_pipeline', 'checking_model_prediction_ratio', 'scheduled__2024-03-16T08:00:00+00:00', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_inference_pipeline/lead_scoring_inference_pipeline.py', '--cfg-path', '/tmp/tmpapecgqb7']
[2024-03-16T09:53:28.681+0000] {standard_task_runner.py:88} INFO - Job 160: Subtask checking_model_prediction_ratio
[2024-03-16T09:53:28.718+0000] {task_command.py:423} INFO - Running <TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-03-16T08:00:00+00:00 [running]> on host daf5d70ef960
[2024-03-16T09:53:28.782+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Lead_scoring_inference_pipeline' AIRFLOW_CTX_TASK_ID='checking_model_prediction_ratio' AIRFLOW_CTX_EXECUTION_DATE='2024-03-16T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-16T08:00:00+00:00'
[2024-03-16T09:53:28.783+0000] {logging_mixin.py:188} INFO - Error while running prediction_ratio_check : Execution failed on sql 'select * from predictions': no such table: predictions
[2024-03-16T09:53:28.784+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-16T09:53:28.787+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Lead_scoring_inference_pipeline, task_id=checking_model_prediction_ratio, execution_date=20240316T080000, start_date=20240316T095328, end_date=20240316T095328
[2024-03-16T09:53:28.813+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-16T09:53:28.835+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
